kt/kt.go

```diff
@@ -17,6 +16,18 @@ import (
 
 const DEFAULT_TIMEOUT = 2 * time.Second
 
+// Error is returned by all functions in this package.
+type Error struct {
+	// Error returned by KT
+	Message string
+	// HTTP status code, if any (0 otherwise)
+	Code int
+}
```

Comment:
I also have an idea about function which tries to cast specified arbitrary error type to KT error:
```
func IsKTError(err interface{}) bool {
    _, ok := err.(Error)

    return ok
}
```

And then I could use it this way:
```
if err != nil && kt.IsKTError(err) {
    return "It's KT failure!"
}
```

What do you think about it?

Reply:
Done